bedrock/firefox/templates/firefox/welcome/page2.html

```diff
@@ -0,0 +1,65 @@
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/welcome/page2" %}
+
+{% extends "firefox/welcome/base.html" %}
+
+{% block page_title %}{{ _('Pocket - Save news, videos, stories and more') }}{% endblock %}
+
+{% block page_image %}{{ static('img/firefox/template/page-image-master.jpg') }}{% endblock %}
+
+{% block body_class %}welcome-page2{% endblock %}
+
+{% block content_intro %}
+  <h1 class="c-page-title">{{ _('Your time online is worth protecting') }}</h1>
+  {# L10n: "Pocket" is a product name and shouldn't be translated. #}
+  <p class="c-page-tagline">{{ _('Discover and save stories in Pocket — and come back to them when you’re free.') }}</p>
+{% endblock %}
+
+{% block content_primary %}
+  <h2 class="body-primary-title"><img src="{{ static('protocol/img/logos/pocket/logo-word-horz.svg') }}" width="186" height="" alt="Pocket"></h2>
+
+  <div class="body-primary-body">
+    <p>{{ _('Pocket is built right into Firefox, so you can easily save stories as you find them, then read them later on any device.') }}</p>
+
+    <img class="primary-image" src="{{ static('img/firefox/welcome/pocket-hero.svg') }}" width="700" height="" alt="">
+
+    <p class="primary-cta">
+      <a id="pocket-cta" class="mzp-c-button mzp-t-product" href="https://getpocket.com/ff_signup?s=ffwelcome2&form_type=button&entrypoint=mozilla.org-firefox-welcome-2&utm_source=mozilla.org-firefox-welcome-2&utm_campaign=welcome-2-pocket&utm_medium=referral" data-action="{{ settings.FXA_ENDPOINT }}" data-cta-text="Activate Pocket" data-cta-type="FxA-Pocket" data-cta-position="Primary" rel="external noopener">{{ _('Activate Pocket') }}</a>
+    </p>
+  </div>
+{% endblock %}
+
+{% block content_secondary %}
+  <div class="c-picto-block t-adjacent-image">
+    <div class="c-picto-block-image">
+      <img src="{{ static('img/icons/pocket-outline-green.svg') }}" alt="">
+    </div>
+
+    <h3 class="c-picto-block-title">{{ _('Save content from everywhere') }}</h3>
+    <div class="c-picto-block-body">
+      <p>{{ _('Grab articles, videos, and links from any website by clicking the Pocket icon in your browser toolbar.') }}</p>
+    </div>
+  </div>
+
+  <div class="c-picto-block t-adjacent-image">
+    <div class="c-picto-block-image">
+      <img src="{{ static('img/icons/navigate-green.svg') }}" alt="">
+    </div>
+
+    <h3 class="c-picto-block-title">{{ _('Discover new stories') }}</h3>
+    <div class="c-picto-block-body">
+      <p>{{ _('Pocket shows recommended stories every time you open a new tab. Save the ones that interest you.') }}</p>
+    </div>
+  </div>
+{% endblock %}
+
+{% block content_utility %}
+  <p><strong><a href="https://support.mozilla.org/kb/firefox-browser-welcome-pages/?utm_source=mozilla.org-firefox-welcome-2&utm_medium=referral&utm_campaign=welcome-2-pocket&entrypoint=mozilla.org-firefox-welcome-2">{{ _('Why am I seeing this?') }}</a></strong></p>
```

Comment:
not a blocker per se; but I think we should make utm params variables at the top of templates. imo, it is a good way to ensure that params are consistent and maintainable. 
For example:
```
{% set _entrypoint = 'mozilla.org-firefox-welcome-2' %}
{% set _utm_campaign = 'welcome-2-pocket' %}
. . . 
<a href="https://support.mozilla.org/kb/firefox-browser-welcome-pages/?utm_source={{ _entrypoint }}&utm_medium=referral&utm_campaign={{ _utm_campaign }}&entrypoint={{ _entrypoint }}">
```